homeassistant/components/google_assistant/__init__.py

```diff
@@ -49,4 +58,24 @@ def async_setup(hass: HomeAssistant, yaml_config: Dict[str, Any]):
     hass.http.register_view(GoogleAssistantAuthView(hass, config))
     hass.http.register_view(GoogleAssistantView(hass, config))
 
+    @asyncio.coroutine
+    def request_sync_service_handler(call):
+        """Handle request sync service calls."""
+        #Code to execute request sync goes here
+        websession = async_get_clientsession(hass) 
+        try:
+            agent_user_id = config.get(CONF_AGENT_USER_ID)
+            with async_timeout.timeout(5, loop=hass.loop):
+                req = yield from session.post(
```

Comment:
You need to do something with the request. I recommend [raise_for_status()](http://aiohttp.readthedocs.io/en/stable/client_reference.html#aiohttp.ClientResponse.raise_for_status) and separately catching and handling `aiohttp.ClientResponseError` and log the error mine looks like this

```python
        except aiohttp.ClientResponseError:
            body = yield from res.read()
            _LOGGER.error(
                'sync request failed: %d %s', res.status, body)
            return
```